pkg/olm/packagemanifest.go

```diff
@@ -5,10 +5,12 @@ import (
 	"fmt"
 	"github.com/golang/glog"
 	pkgManifestV1 "github.com/operator-framework/operator-lifecycle-manager/pkg/package-server/apis/operators/v1"
+	v1 "github.com/operator-framework/operator-lifecycle-manager/pkg/package-server/apis/operators/v1"
```

Comment:
Use `pkgManifestV1` from the line above. Besides, `v1` could be many different things so hijacking it for pkg manifests would be a bad idea :)

Reply:
you are right I missed that, fixing now